dockerfiles/trellis/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM trellisldp/trellis
+
```

Comment:
I'd recommend adding a few environment values to trellis so that its behavior conforms to the expectations of the LDP test suite:

```
ENV JAVA_OPTS="-Dtrellis.http.weak.etag=false -Dtrellis.http.memento.headerdates=false -Dtrellis.http.precondition.required=true"
```